packages/react-component-library/.storybook/preview.js

```
@@ -1,6 +1,33 @@
 import '@royalnavy/css-framework/index.scss'
 import '@royalnavy/fonts'
 
+/**
+ * Hacky way of clicking on Docs button on first load of page.
+ * https://github.com/storybookjs/storybook/issues/13128
+ *
+ */
+function clickDocsButtonOnFirstLoad() {
+  window.removeEventListener('load', clickDocsButtonOnFirstLoad)
+
+  try {
+    const docsButtonSelector = window.parent.document.evaluate(
+      "//button[contains(., 'Docs')]",
+      window.parent.document,
+      null,
+      XPathResult.ANY_TYPE,
+      null
+    )
+
+    const button = docsButtonSelector.iterateNext()
+
+    button.click()
+  } catch (error) {
+    // Do nothing if unable to click on Docs button
```

Can we do anything here rather than swallowing the exception?